app/helpers/mailer_helper.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module MailerHelper
+
+  def determine_logo_url
+    host      = confirmation_url(Account.first) # TODO: need way to determine a host correctly, just tip to me, but it works!
+    websites  = Website.all
```

Comment:
we should compare current application domain name and find it in website
we may want to make a fallback variable like APP_DOMAIN